flake.nix

```diff
@@ -78,7 +78,7 @@
 
         pkgs = import np {
           inherit system config;
-          overlays = [ overlay ds.overlay ] ++ shs.overlays."${system}";
```

Comment:
This needs to be added again at some point, but is broken right now due to souffle-haskell pointing to GHC 9.2.2 and we are stuck on 8.X.X due to llvm-hs (for now).

Reply:
Heh, apparently the CI is broken because of it.. the LLVM issue just got top priority :smirk:.